devel/chronos-go/Portfile

```
@@ -0,0 +1,72 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/amit-davidson/Chronos .0.1.0 v
```

Looks like this makes the port have the version number ".0.1.0". Should that be "0.1.0" instead?
```suggestion
go.setup            github.com/amit-davidson/Chronos 0.1.0 v.
```

---

Apologies @ryandesign - resolved